unittests/Swift/BuildDBBindingsTests.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  BuildDBBindingsTests.swift
+//  llbuildSwiftTests
+//
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+import XCTest
+
+// The Swift package has llbuildSwift as module
+#if SWIFT_PACKAGE
+import llbuild
+import llbuildSwift
+#else
+import llbuild
+#endif
+
+class BuildDBBindingsTests: XCTestCase {
+  
+  let exampleBuildDBClientSchemaVersion: UInt32 = 9
+  
+  var exampleBuildDBPath: String {
+    return URL(fileURLWithPath: #file)
+      .deletingLastPathComponent()
+      .appendingPathComponent("Resources", isDirectory: true)
+      .appendingPathComponent("build.db")
+      .path
+  }
+  
+  func exampleDB() throws -> BuildDB {
+    let newPath = "\(self.tmpDirectory!)/build.db"
```

Comment:
I'm a little worried about the maintenance overhead of including an example db for this test.  As your own comments attest, our db versioning strategy currently is to merely delete incompatible versions and move on. We don't maintain compatibility with older schemas.

It would be preferable to actually generate the test build db programatically.

Reply:
Ah, that's a good idea 👍

Reply:
I added a function that generates the database by basically doing similar things as the Swift example in the project.